README.md

```diff
@@ -37,18 +37,94 @@ The latest version from the release branch 'master' is automatically deployed to
 
 $ `git clone https://github.com/ita-social-projects/ZeroWaste.git`
 
-## Setup
+## Local setup
+First of all you need rvm to setup project. For the operating system Windows the optimal solution is to use <a href="https://docs.microsoft.com/en-us/windows/wsl/">WSL</a>.
 
 $ `bin/setup`
-or 
-$ `bundle install` 
+or
+$ `bundle install`
+
+<b>PostgreSQL</b>
+
+<a href="https://www.postgresql.org/download/">Install PostgreSQL</a> for your operating system or subsystem.
+You can familiarize yourself with <a href="https://www.postgresql.org/docs/">PostgreSQL documentation</a>.
+
+In your local machine in cloned project in config folder rename database.yml.sample to database.yml. Make sure that the user and password match the data in this file. Port may be changed.
+
+<b>pg gem</b>
+
+Under certain circumstances bundle can do not install pg.
+
+To install manually:
+$ `sudo apt-get install libpq-dev`
+then
+$ `gem install pg`
+
+<b>Database configure</b>
+
+For correct operation of the migration, you need to rename the migration file `20220123171144_create_versions.rb` so that it is processed first.
+
+To create the necessary databases and update them:
+
```

Comment:
Also adding rake db:reset first would help if someone is having troubles with the database.